main.nf

```diff
@@ -119,6 +121,8 @@ if (!params.skip_demultiplexing) {
     summary['Barcode Kit ID']     = params.barcode_kit ?: 'Unspecified'
     summary['Guppy Config File']  = params.guppy_config ?: 'Unspecified'
     summary['Guppy GPU Mode']     = params.guppy_gpu ? 'Yes' : 'No'
+    summary['Guppy GPU Runners']  = params.guppy_gpu_runners ?: 'Unspecified'
+    summary['Guppy CPU Threads']  = params.guppy_cpu_threads ?: 'Unspecified'
```

Comment:
```suggestion
    summary['Guppy CPU Threads']  = params.guppy_cpu_threads
```

Reply:
These parameters have default values set so we can just print those, and if they have been customised then those will be printed instead.